ament_index_python/ament_index_python/search_paths.py

```diff
@@ -14,10 +14,12 @@
 
 import os
 
+from typing import List, Union
+
 from .constants import AMENT_PREFIX_PATH_ENV_VAR
 
 
-def get_search_paths():
+def get_search_paths() -> Union[List[str], List[os.PathLike[str]]]:
```

Comment:
It seems a little weird to me that this is a Union; it looks like it should just be `List[str]`, given the code.  Could you explain more about why you chose this type?